axon-application-development/command-handling/modeling/conflict-resolution.md

```diff
@@ -0,0 +1,16 @@
+# Conflict Resolution
```

Comment:
This is the second time I see this file in this PR. Has some stuff been duplicated for the moment per chance?